raster3d/r3.in.ascii/r3.in.ascii.html

```diff
@@ -106,7 +102,7 @@ <h2>SEE ALSO</h2>
 <a href="g.region.html">g.region</a>
 </em>
 
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
+<h2>AUTHORS</h2>
+
+Roman Waupotitsch, Michael Shapiro, 
+Helena Mitasova, Bill Brown, Lubos Mitas, Jaro Hofierka, S&ouml;ren Gebbert
```

Comment:
Duplicate `<h2>NOTES</h2>` (line 14, 28) and `<h2>EXAMPLES</h2>` (line 61, 91) header, is it ok?

Reply:
Good catch, I have replaced it with `<h3>Format</h3>`